WinrtServer/Programmer.idl

```diff
@@ -6,6 +6,11 @@ namespace WinrtServer
         Single z;
     };
 
+    struct Favorites {
+        String FavoriteDrink;
+        String FavoriteActivity;
```

Comment:
Readability nitpick: You probably don't need  the "Favorite" prefix in the members, since the surrounding struct is already named Favorites.